code/game/mecha/mech_fabricator.dm

```diff
@@ -1,3 +1,10 @@
+
+#define MECHFAB_MAIN_MENU		1
+#define MECHFAB_CATEGORY_MENU	2
+#define MECHFAB_SEARCH_MENU		3
+#define MECHFAB_MATERIALS_MENU	4

```

Comment:
I suggest using strings instead of numbers here. AFAIK comparing those is just as cheap, but you get a tiny bit of debug info that way.